### PR TITLE
docs: add temporary website for docs contribution

### PR DESCRIPTION
**Pull request by atinux**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)

### 📚 Description

Add a temporary `.website/` folder using Docus until we can open source `nuxt.com` (currently rewritten using NuxtLabs UI) for everyone to easily contribute.

So far it is missing components compared to Docus but it is already much better in development to contribute to docs than in the current state.

Command to run: `pnpm docs`

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .website/.npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+shamefully-hoist=true
+strict-peer-dependencies=false
```
Comment (danielroe) [inline .website/.npmrc:2]: do we need this?

File: .website/nuxt.config.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    sources: {
+      docs: {
+        driver: 'fs',
+        prefix: '/',
+        base: resolve('../docs')
+      }
+    }
+  },
+  experimental: {
+    renderJsonPayloads: false
```
[the remark this answers is not in the document]
Comment (Atinux) [reply .website/nuxt.config.ts:18]: Very weird but having it was breaking the client-side hydration, could you confirm on your end?

File: .website/.npmrc (hunk 1 above)
Comment (Atinux) [reply .website/.npmrc]: Removed

**Review by danielroe — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "lint:docs": "markdownlint ./docs && case-police 'docs/**/*.md' *.md",
     "lint:docs:fix": "markdownlint ./docs --fix && case-police 'docs/**/*.md' *.md --fix",
     "lint:knip": "pnpx knip",
+    "docs": "nuxi dev .website",
```
Comment (danielroe) [inline package.json:17]:
```suggestion
    "docs:dev": "nuxi dev .website",
```

File: .website/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+```bash
+npm run build:stub
+```
+
+## Development
+
+In the root of the `nuxt` folder, run:
+
+```bash
+npm run docs
```
Comment (danielroe) [inline .website/README.md:26]:
```suggestion
pnpm docs:dev
```

File: .website/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Install dependencies in the root of the `nuxt` folder:
+
+```bash
+pnpm i
+```
+
+Then stub the dependencies:
+
+```bash
+npm run build:stub
```
Comment (danielroe) [inline .website/README.md:18]:
```suggestion
pnpm build:stub
```

File: .website/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Nuxt Docs Website
+
+This is a tempory directory until we open source the repository for nuxt.com.
```
Comment (danielroe) [inline .website/README.md:3]:
```suggestion
This is a temporary directory until we open source the repository for nuxt.com.
```

**Comment by Barbapapazes** [comment]
does not work if `npm run build` is not run before using `npm run docs:dev`.

**Pull request merged**